src/host/usbh.c

```diff
@@ -655,7 +655,9 @@ static bool usbh_control_xfer_cb (uint8_t dev_addr, uint8_t ep_addr, xfer_result
           TU_ASSERT( hcd_edpt_xfer(rhport, dev_addr, tu_edpt_addr(0, request->bmRequestType_bit.direction), _ctrl_xfer.buffer, request->wLength) );
           return true;
         }
+#if __GNUC__ >= 7
```

Comment:
we still need to tell gcc 6 this is intended fallthough, also since usbh.c is compiled with other gcc we should check if GNUC is defined first at well